Identity/Identity.API/Helpers/ApplicationSettingsHelper.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Identity.API.Helpers;
+
+public class ApplicationSettingsHelper
```

Comment:
Singleton for configuration is not the way to go.

I see you're mostly using this for the `JWTHelper`, which has a `static` method for the generation. What would be better in this case is to have a non static JWT generation function/class, e.g. `JwtGenerator`, which would be injected where needed. Configuration would be injected into it, ideally using [options pattern](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/configuration/options?view=aspnetcore-8.0#the-options-pattern). 

Would also make these classes more testable

Reply:
Changed to use DI.